lib/ui/profile/Profile.dart

```diff
@@ -250,16 +269,16 @@ class UserProfile extends StatelessWidget {
     );
   }
 
-  Widget _buildActiveCrops() {
+  Widget _buildActiveCrops(ProfileViewModel viewModel) {
     return Column(
       crossAxisAlignment: CrossAxisAlignment.start,
       children: <Widget>[
         Text(
-          _viewModel.activeCrops.toString(),
+          viewModel.activeCrops.toString(),
           style: _style.subtitleTextStyle,
         ),
         Text(
-          _LocalisedStrings.activeCrops(),
+          Intl.message(_LocalisedStrings.activeCrops()),
```

Comment:
_LocalisedStrings.activeCrops() is already an Intl.message :)

```suggestion
          _LocalisedStrings.activeCrops(),
```

Reply:
ah missed this in merge

Reply:
np!

Reply:
👍